### PR TITLE
[CNSMR-2925] Add tutorial for accessing App Center builds

### PR DESCRIPTION
**Pull request by MichalTKwiecien**
This is a document describing how App Center builds can be accessed since we're moving from HockeyApp to App Center.

### CODE REVIEW

**Review by karabatov — APPROVED** [review]
👍

File: Cookbook/Technical-Documents/AppCenter.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Accessing App Center builds
+
+### Steps to install
+1. On your device, visit [App Center](http://appcenter.ms/apps)
+
+2. As a login method, select `Company or school`
+
+	![app-center-company-or-school](./Assets/app-center/company-or-school.png)
```
Comment (dmcrodrigues) [inline Cookbook/Technical-Documents/AppCenter.md]: The images are not loading, the reference is probably wrong, maybe `Assets/app-center/company-or-school.png`?

**Review by dmcrodrigues — APPROVED** [review]

File: Cookbook/Technical-Documents/AppCenter.md (hunk 1 above)
Comment (karabatov) [reply Cookbook/Technical-Documents/AppCenter.md]: They aren't working in the diff, you need to select “View file” from the dropdown menu for them to work :)

Comment (dmcrodrigues) [reply Cookbook/Technical-Documents/AppCenter.md]: :disappear: 😆

**Pull request merged**